functions/Install-DbaMaintenanceSolution.ps1

```diff
@@ -230,7 +232,7 @@ function Install-DbaMaintenanceSolution {
 
         function Get-DbaOlaWithParameters($listOfFiles) {
 
-            $fileContents = @{ }
+            $fileContents = [ordered] @{}
```

Comment:
```suggestion
            $fileContents = [ordered]@{ }
```
That should make our format test pass.  OTSB requires the space with brackets...if I recall.